apps/sms/js/activity_handler.js

```diff
@@ -241,7 +246,11 @@ var ActivityHandler = {
         return;
       }
 
-      Utils.confirm('discard-new-message').then(() => {
+      Utils.confirm(
+        'discard-new-message',
+        'unsent-message-title',
```

Comment:
Maybe same for similar case is better? I don't see much benifit if we differentiate the discard confrim here.